assets/js/components/link.js

```diff
@@ -25,9 +25,24 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import { _x } from '@wordpress/i18n';
 import { Component } from '@wordpress/element';
 
 class Link extends Component {
+	getArialLabel() {
+		const {
+			children,
+			external,
+		} = this.props;
+
+		let label;
+		if ( external && typeof children === 'string' ) {
+			label = `${ children } ${ _x( '(opens in a new tab)', 'screen reader text', 'google-site-kit' ) }`;
+		}
```

Comment:
This should respect an [`aria-label` prop](https://reactjs.org/docs/accessibility.html#wai-aria) if passed, and only use `children` as a fallback (see other comments below).

I.e.
- if `aria-label`, append the new tab text
- if no `aria-label`, but string `children`, append the new tab text to `children`
- if no `aria-label` or string `children` return the new tab text

Reply:
@aaemnnosttv Updated. Can you check?